docs/tutorials/tut-setup-dev-codespace.md

```diff
@@ -0,0 +1,97 @@
+using a [dev container](https://docs.github.com/en/Codespaces/setting-up-your-project-for-Codespaces/adding-a-dev-container-configuration/introduction-to-dev-containers).
+
+Your Codespace environment will be hosted on GitHub's servers (attached to your GitHub account),
+and will allow you to access your Codespace from any computer, continuing your work from where you left off.
+
+### Cost and Limitations
+GitHub offers a free quota for Codespaces (which is higher for GitHub Pro users) that you can utilize for developing Cortex XSOAR content.  
+
+The quota is calculated based on the number of hours your Codespace is actively running.
+
+:::info
+Codespaces generated from our Content repository (or a fork of it) use 4-cores by default.
+:::
+
+You can see the free quota plan and additional information [here](https://docs.github.com/en/billing/managing-billing-for-github-Codespaces/about-billing-for-github-Codespaces#monthly-included-storage-and-core-hours-for-personal-accounts).  
+
+:::tip
+You can find information about your Codespace quota usage on the settings under "Billing and plans".  
+See the following GitHub article for a step-by-step guide: [Viewing your GitHub Codespaces usage](https://docs.github.com/en/billing/managing-billing-for-github-Codespaces/viewing-your-github-Codespaces-usage).
+
+You will receive an email notification when you have used 75%, 90%, and 100% of your freely-included quotas.
+:::
+
+If you want to use paid usage once you've reached your free quota, you can find information about the pricing [here](https://docs.github.com/en/billing/managing-billing-for-github-Codespaces/about-billing-for-github-Codespaces#pricing-for-paid-usage).
+
+## Requirements
+A GitHub account (can be created [here](https://github.com/signup)).
+
+## Setup
+The following provides instructions for an initial setup that needs to be done only once.  
+After that, the generated environment will be kept in your GitHub account.
+
+### Create a New Codespace
+1. [Log in to your GitHub account](https://github.com/login) (if not already logged in).
+2. Enter the [XSOAR Content repository](https://github.com/demisto/content).
+3. Fork the repository to your account:
+   1. Click **Fork** at the top right.
+   2. Select your account as the owner, and leave the repository name as is.
+   3. Keep the **Copy the master branch only** option selected.
+   4. Click **Create fork**.
+4. Wait a few seconds for the fork to be created. You will be redirected to your forked repository page.
+5. Create a new branch on your fork, and give it a meaningful name.
+6. Click **Code**, go to the **Codespaces** tab, and click **Create Codespace on \<branch name\>**.
+7. A page saying "This Codespace is requesting additional permissions" might appear.  
+    If it does, click **Continue without authorizing**.
+8. Click **New Codespace** (this might take a few minutes).
```

Comment:
When I tried it, it created the Codespace for me but was stuck in `loading` page.
It's worth adding to go to the repo and open the codespace from there